cli/cmd/install.go

```diff
@@ -234,9 +235,22 @@ func doInstallation() error {
 
 	logging.PrintLog("Keptn has been successfully set up on your cluster.", logging.InfoLevel)
 	logging.PrintLog("---------------------------------------------------", logging.InfoLevel)
-	fmt.Println("* To quickly access Keptn, you can use a port-forward and then authenticate your Keptn CLI (in a Linux shell):\n" +
-		" - kubectl -n " + keptnNamespace + " port-forward service/api-gateway-nginx 8080:80\n" +
-		" - keptn auth --endpoint=http://localhost:8080/api --api-token=$(kubectl get secret keptn-api-token -n " + keptnNamespace + " -ojsonpath={.data.keptn-api-token} | base64 --decode)\n")
+
+	if installParams.EndPointServiceType.String() == "NodePort" || installParams.EndPointServiceType.String() == "LoadBalancer" {
+		endpoint, err := getAPIEndpoint(keptnNamespace, installParams.EndPointServiceType.String())
+		apiToken, _ := getAPITokenFromSecret(keptnNamespace)
+		if err == nil {
+			fmt.Println("* To quickly access Keptn, you can authenticate your Keptn CLI (in a Linux shell):\n" +
+				" - keptn auth --endpoint=" + endpoint + " --api-token=" + apiToken + "\n")
+		} else {
+			errorFlag = true
+		}
+	}
+	if errorFlag {
```

Comment:
What if `EndPointServiceType` is ClusterIP? We'll never see the message that's now hidden behind `if errorFlag`.

I recommend to turn this around by renaming  `errorFlag` into `showFallbackConnectMessage`, and set `showFallbackConnectMessage` to `false` when you have successfully fetched an endpoint and port.

Reply:
Thanks for raising this, my bad, I was thinking to implement this the other way around and ended up doing this. will fix this.